src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -328,6 +329,19 @@
     <formal-name>Adjustment Justification</formal-name>
     <description>If the selected security level is different from the base security level, this contains the justification for the change.</description>
   </define-field>
+  <define-flag name="characterization-ns" as-type="uri">
+    <formal-name>Characterization Namespace</formal-name>
+    <description>A namespace qualifying the system information characterization scheme.</description>
+    <use-name>ns</use-name>
+    <constraint>
+      <allowed-values allow-other="yes">
+        <enum value="http://csrc.nist.gov/ns/fips-199">The system is categorized according to <a href="https://csrc.nist.gov/pubs/fips/199/final">FIPS-199.</a></enum>
+      </allowed-values>
+    </constraint>
+    <remarks>
+       <p>This value must be an <a href="https://pages.nist.gov/OSCAL/concepts/uri-use/#absolute-uri">absolute URI</a> that serves as a <a href="https://pages.nist.gov/OSCAL/concepts/uri-use/#use-as-a-naming-system-identifier">naming system identifier</a>.</p>
+    </remarks>
+ </define-flag>
```

Comment:
What does this afford us that using the normal `@ns` extension mechanism doesn't?

Reply:
Re our discussion on gitter, the idea would be that the `@ns` would apply to the entire `system-categorization`, which would allow for future (in this PR or in future PRs) constraints to apply to the `information.types.*-impact`, `security-impact-level.*`, and `security-sensitivity-level`

Reply:
OK, without grouping as a backwards compatible option, I would recommend we stick to `@ns` and not add this at this time lest we make things more complex and hard to manage. If I misunderstanding something, let's chat (I guess after review, or during if you're up to it).